src/include/basil.h

```diff
@@ -397,7 +469,7 @@ typedef enum {
 	basil_memory_type_none = 0,
 	basil_memory_type_os,
 	basil_memory_type_hugepage,
-	basil_memory_type_virtual
+    basil_memory_type_virtual
```

Comment:
Please double check all whitespace in the file to ensure tabs and spaces are used appropriately.